specification/sql/resource-manager/readme.md

```diff
@@ -268,7 +268,7 @@ input-file:
 - Microsoft.Sql/stable/2014-04-01/serviceObjectives.json
 - Microsoft.Sql/stable/2014-04-01/sql.core.json
 - Microsoft.Sql/stable/2014-04-01/usages.json
-- Microsoft.Sql/preview/2015-05-01-preview/blobAuditingPolicies.json
+- Microsoft.Sql/preview/2015-05-01-preview/blobAuditing.json
 - Microsoft.Sql/preview/2015-05-01-preview/databaseAutomaticTuning.json

```

Comment:
Are you sure that tag `package-2017-03-preview` should contain 2 versions of `blobAuditing.json`? 2015-05-01-preview and 2017-03-01-preview?